digital_land/cli.py

```diff
@@ -291,6 +293,24 @@ def transform_cmd(input_path, output_path):
     save(stream, output_path, SPECIFICATION.current_fieldnames(PIPELINE.schema))
 
 
+@cli.command("load_entries", short_help="load_entries")
+@click.option("--output-path", type=click.Path(), default=None)
+@click.argument("input-paths", nargs=-1, type=click.Path(exists=True))
+def load_entries_cmd(input_paths, output_path):
+    if not output_path:
+        print("missing output path")
+        sys.exit(2)
+
+    repo = EntryRepository(output_path, create=True)
+    loader = EntryLoader(repo)
+
+    total = len(input_paths)
+    for idx, path in enumerate(input_paths, start=1):
+        logging.info("loading file %s of %s", idx, total)
+        stream = load_csv_dict(path, line_num=True)
```

Comment:
`line_num=True` --> `include_line_num=True`

Reply:
good spot - I changed the name half way through development. I wonder why this doesn't break!

Reply:
It did haha. I noticed it when trying to run it locally

Reply:
ah damn. Pretty much the only bit of code not covered by a unit test. This is the kind of plumbing that an e2e test would catch. Or we'd have to mock the EntryLoader and check it's called properly from cli.py.